discussment-core/src/main/java/org/danekja/discussment/core/domain/PostReputation.java

```diff
@@ -0,0 +1,89 @@
+package org.danekja.discussment.core.domain;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.danekja.discussment.core.domain.PostReputation.GET_BY_POST;
+
+/**
+ * The class represents the post's reputation on the website
+ *
+ * Date: 16.2.18
+ *
+ * @author Jiri Kryda
+ */
+@Entity
```

Comment:
Turn this into `@Embeddable` instead of `@Entity` to get rid of unecessary table in the relation. Kills performance.

[https://www.objectdb.com/api/java/jpa/Embeddable](https://www.objectdb.com/api/java/jpa/Embeddable)